src/actions/ticket-actions.js

```diff
@@ -483,7 +483,11 @@ export const getTickets =
       per_page: perPage,
       access_token: accessToken,
       expand:
-        "owner,order,ticket_type,badge,badge.type,promo_code,promo_code.tags,refund_requests,refund_requests.refunded_taxes"
+        "owner,ticket_type,badge,badge.type,promo_code,promo_code.tags,refund_requests",
```

Comment:
same for ticket type
u only need the id and look for the type on the current summit